app/src/main/res/values/example_descriptions.xml

```diff
@@ -84,7 +84,8 @@
     <string name="description_large_geojson">Display long route as large geojson</string>
     <string name="description_nine_patch">Add 9-patch image to the style</string>
     <string name="description_feature_state">Create interactive hover effects with feature state</string>
-    <string name="description_3d_globe">Showcasing 3D globe</string>
+    <string name="description_3d_globe">Create a map using the globe projection.</string>
+    <string name="description_spinning_globe">Display your map as an interactive, rotating globe.</string>
```

Comment:
Also here - maybe align `display` / `create`

Reply:
was aligned with gl-js, not my texting 😄